src/components/SVGViewer/stories/basic.story.mdx

```diff
@@ -0,0 +1,102 @@
+import { Meta, Story, Preview, Props, Source } from '@storybook/addon-docs/blocks';
+import { SVGViewer } from '../SVGViewer';
+import { decorators } from './helper';
+
+<Meta title='Other|SVGViewer' decorators={decorators} />
+
+# SVG-viewer
+
+SVG-viewer is a component which visualise `.svg`-documents in a user interactive way.
+
+SVG-viewer provides the following functionality:
+- Display documents with zoom and panning functionality for both desktop and mobile
+- Highlight equipment and provide callbacks for further interaction
+- Locate and zoom onto equipment in the document
+- Search by equipment with locating and zooming results
+
+**NOTE:** The component should have `ClientSDKProvider` as a parent component in react component tree.
+
+## Basic Usage:
+
+Imports:
+
+```typescript jsx
+import 'antd/dist/antd.css';
+
+import React from 'react';
+import { SVGViewer } from '@cognite/gearbox';
+```
+
+<Preview>
+  <Story name='Basic Usage'>
+<div style={{height: '100vh'}}>
+    <SVGViewer documentId={5185355395511590} />
+</div>
+  </Story>
+</Preview>
+
+## Available props
+
+<Props of={SVGViewer} />
+
+## Types
+
+### Conditions
+
+This type describes the parameters the `metadataClassesConditions` function is called with.
+The type can be imported from `@cognite/gearbox`:
+
+```typescript
```

Comment:
`typescript` is also not supported yet, use `js` instead

Reply:
fixed